lib/elixir/lib/calendar.ex

```diff
@@ -172,6 +172,16 @@ defmodule Calendar do
   `starting_on` represents the starting day of the week. All
   calendars must support at least the `:default` value. They may
   also support other values representing their days of the week.
+
+  The value of `day_of_week` is an ordinal number meaning that a
+  value of `1` is defined to mean "first day of the week". It is
+  specifically not defined to meean `1` is `Monday`.
+
+  It is a requirement that `first_day_of_week` is less than `last_day_of_week`
+  and that `day_of_week` must be within that range. Therefore it can be said
+  that `day_of_week in first_day_of_week..last_day_of_week//1` must be
+  `true` for all values of `day_of_week`.
+
```

Comment:
```suggestion
  specifically not defined to mean `1` is `Monday`.

  It is a requirement that `first_day_of_week` is less than `last_day_of_week`
  and that `day_of_week` must be within that range. Therefore it can be said
  that `day_of_week in first_day_of_week..last_day_of_week//1` must be
  `true` for all values of `day_of_week`.
```